packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -57,6 +60,11 @@ const nodesCountDeep = [
 	[100, BenchmarkType.Measurement],
 ];
 
+const factory = new SharedTreeFactory({
+	jsonValidator: typeboxValidator,
+	treeEncodeType: TreeCompressionStrategy.Uncompressed,
```

Comment:
why uncompressed? this is a benchmark suite :)

Reply:
This test also had missing/incorrect schema, so I updated with a TODO comment with the ADO task